### PR TITLE
At/#1886 error messages wording

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
change wording of some messages
simplify markup of warning alert
add some stuff to storybook

### CODE REVIEW

File: app/components/settings/backup-seed/BackupSeedVerify.tsx [hunk 1]
```diff
@@ -18,6 +13,12 @@ import { WarningAlert } from "../../shared/WarningAlert";
 import * as arrowLeft from "../../../assets/img/inline_icons/arrow_left.svg";
 import * as styles from "./BackupSeedVerify.module.scss";
 
+/* tslint:disable: no-submodule-imports */
+import "react-select/dist/react-select.css";
+import "react-virtualized-select/styles.css";
+import "react-virtualized/styles.css";
+/* tslint:enable: no-submodule-imports */
+
```
Comment (AlexeyTuboltsev) [inline app/components/settings/backup-seed/BackupSeedVerify.tsx]: i'm fixing  CSS compilation precedence here

File: app/components/settings/backup-seed/BackupSeedVerify.tsx [hunk 2]
```diff
@@ -161,36 +157,30 @@ class BackupSeedVerify extends React.Component<IBackupSeedVerifyProps, IBackupSe
...
+            iconPosition="icon-before"
+            svgIcon={arrowLeft}
+            onClick={this.props.onBack}
+          >
+            <FormattedMessage id="form.button.back" />
+          </Button>
+        </div>
       </>
     );
   }
```
Comment (AlexeyTuboltsev) [inline app/components/settings/backup-seed/BackupSeedVerify.tsx]: simplify markup: get rid of 1000+ Row/Col in favor of a Div with proper styling

File: app/components/shared/WarningAlert.tsx [hunk 3]
```diff
@@ -9,13 +8,10 @@ interface IWarningAlertProps {
...
-    <Col xs={10}>
+  <div className={cn(styles.warningAlert, className)}>
+    <span>
       <i className={cn("fa fa-exclamation-circle mr-2", styles.icon)} aria-hidden="true" />
       {children}
-    </Col>
-  </Row>
+    </span>
+  </div>
 );
```
Comment (AlexeyTuboltsev) [inline app/components/shared/WarningAlert.tsx]: simplify markup

File: app/index.tsx [hunk 4]
```diff
@@ -39,6 +34,11 @@ import * as ga from "./utils/googleAnalitycs.js";
 import { IntlProviderAndInjector } from "./utils/IntlProviderAndInjector";
 import { InversifyProvider } from "./utils/InversifyProvider";
 
+import "font-awesome/scss/font-awesome.scss";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import "./styles/overrides.scss";
+
```
Comment (AlexeyTuboltsev) [inline app/index.tsx]: this is to fix  CSS compilation precedence: bootstrap goes first, than all imports inside of modules, than some overrides and imported styles

File: app/components/dashboard/my-portfolio/MyPortfolioWidget.stories.tsx [hunk 5]
```diff
@@ -21,4 +21,11 @@ storiesOf("MyPortfolioWidget", module)
       isLoading={false}
       {...{ balanceEur: "0", balanceNeu: "0", isIcbmWalletConnected: true }}
     />
+  ))
+  .add("error", () => (
+    <MyPortfolioWidgetComponent
+      isLoading={false}
+      error={"bla bla error"}
```
Comment (huesforalice) [inline app/components/dashboard/my-portfolio/MyPortfolioWidget.stories.tsx]: In general I would recommend putting a longer text here, as there will probably be a longer text in reality also. This is the only way you can test the widget styles properly

File: app/components/settings/kyc-states/KycStatusWidget.stories.tsx [hunk 6]
```diff
@@ -70,4 +70,12 @@ storiesOf("KYC/StatusWidget", module)
       requestStatus="Outsourced"
       requestOutsourcedStatus="review_pending"
     />
+  ))
+  .add("error", () => (
+    <KycStatusWidgetComponent
+      {...{ ...commonProps, error: "bla bla error", loading: true }}
```
Comment (huesforalice) [inline app/components/settings/kyc-states/KycStatusWidget.stories.tsx]: same comment about previewing texts as above

**Pull request merged**